metabase/metabase
# use eval-source-map when BETTER_SOURCE_MAPS is set

### Description

Source maps are not working well even when using `BETTER_SOURCE_MAPS` env variable. This PR switches to `eval-source-map` which is better than `inline-source-map` even in terms of [rebuild performance](https://webpack.js.org/configuration/devtool/).

### How to verify

1. Run `BETTER_SOURCE_MAPS=true yarn dev`
2. Try to debug any file and ensure you can see the original file sources and **add breakpoints anywhere it should be possible**

### Demo

#### Before

https://user-images.githubusercontent.com/14301985/221441524-f13210f9-c512-4875-8694-522b9b332d24.mov

#### After

https://user-images.githubusercontent.com/14301985/221441530-c25a0280-ec10-453d-a262-d5574b38c4d0.mov

### Checklist

- [n/a] Tests have been added/updated to cover changes in this PR — config change
